api/routes/members.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express'
+import User from '../models/User'
+
+const router = express.Router()
+
+router.get('/:id', (req, res) => {
+  User.find({ _id: req.params.id }, (err, boards) => {
+    if (err) {
+      res.send(err)
+    } else {
+      res.json(boards)
+    }
+  })
+})
+
+// For tests mainly
+router.get('/initials/:initials', (req, res) => {
```

Comment:
Even it's for test, I don't understand this route. You pass initials, you search in the the User collection and you return some boards ? Why ?

Reply:
Missed a part in my copy paste :P